stage2/tasks/fun-chat/README.md

```diff
@@ -154,3 +166,7 @@ _Answer_ - It is not considered a layout issue if only the first 10 characters o
 _Question_ - If a user sends a very long message, how can the requirements of the interface points be met?
 
 _Answer_ - The complete message must be displayed in the dialogue.
+
+_Question_ - What pages are available to a non-authorized user?
+
+_Answer_ - It has access to the authentication and about pages.
```

Comment:
_Answer_ - They have access to the Authentication and About pages.

Reply:
done